src/content/en/2020/seo.md

```diff
@@ -9,70 +8,72 @@ reviewers: [natedame, catalinred, dsottimano, dwsmart, en3r0, ibnesayeed, bazzad
+  description="Screenshot from Google Trends showing the change in time for three three digital marketing related terms: Search Engine Optimization starts at 25% but shows an increasing importance over time to 75% now. Pay-per-click similarly starts at 25% but started falling in 2009 and looks to be less than 10% now. Social-media marketing starts very small (approximately 5%) and shows some slight growth over time ending up slightly above pay-per-click around the 10% mark.",
   width=1600,
   height=844
   )
 }}
 
-The goal of the Web Almanac’s SEO chapter is to identify and assess main elements and configurations that play a role in a website’s organic search optimization. By identifying these elements we hope that websites can leverage our findings to improve their ability to be crawled, indexed, and ranked by search engines. In this chapter we provide a snapshot of their status in 2020 and a summary of what has changed [since 2019](../2019/seo).
+The goal of the Web Almanac's SEO chapter is to identify and assess main elements and configurations that play a role in a website's organic search optimization. By identifying these elements, we hope that websites can leverage our findings to improve their ability to be crawled, indexed, and ranked by search engines. In this chapter, we provide a snapshot of their status in 2020 and a summary of what has changed [since 2019](../2019/seo).
 
-It is important to note that besides the [HTTP Archive](https://httparchive.org/) this analysis also includes data from [Lighthouse](https://developers.google.com/web/tools/lighthouse/), the [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report), as well as raw and rendered HTML elements. In the case of the HTTP Archive and Lighthouse it is limited to the data identified from websites’ home pages only, not site-wide crawls. We have taken this into consideration when doing assessments. Keeping this distinction in mind is important when drawing conclusions from our results. You can learn more about it in our [methodology](./methodology) page.
+It is important to note that this chapter is based on analysis from [Lighthouse](https://developers.google.com/web/tools/lighthouse/), the [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report), as well as raw and rendered HTML elements from the [HTTP Archive](https://httparchive.org/) crawl. In the case of the HTTP Archive and Lighthouse, it is limited to the data identified from websites' home pages only, not site-wide crawls. We have taken this into consideration when doing assessments. Keeping this distinction in mind is important when drawing conclusions from our results. You can learn more about it on our [Methodology](./methodology) page.
 
-Let us go through this years’ websites Organic Search optimization main findings.
+{# TODO(authors): Is "Organic Search" a proper noun? Or should it be lowercase? #}
```

Comment:
@aleyda

Reply:
@rviscomi should be lowercase! Should I directly update the doc? Just let me know the best way to proceed to fix :) Thanks

Reply:
@rviscomi saw your comment below about directly fixing and pushing commits so I already replaced this! Thanks